test/integration/test_expid.py

```diff
@@ -15,14 +15,33 @@
 # You should have received a copy of the GNU General Public License
 # along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>.
 
-"""Tests for ``autosubmit expid``."""
+"""
+    Tests for ``autosubmit expid``.
+    Tests for experiment IDs.
```

Comment:
I think we can keep just one of these comments? :point_up: The spaces are also not needed. Normally they are

`"""Comment here, no spaces around it"""`

Or

```
"""
Comment here, no spaces either
"""
```